api/routes/users.js

```diff
@@ -5,18 +5,96 @@ const jwt = require('jsonwebtoken');
 const { json } = require('express');
 const auth = require('../middleware/auth');
 const mailService = require('../middleware/mailService');
+const owasp = require('owasp-password-strength-test');
 
 const jwtSecret = process.env.SECRET || 'devsecret';
 const sessionTimeout = process.env.SESSION_TIMEOUT;
 
-let createUser = async ({ username, email, organization, role }) => {
-  let user = new User({ email, username, organization, role });
-  await user.save();
-  let emailSubject = 'Responsible AI Design Assistant Account Creation';
-  let emailTemplate = 'api/emailTemplates/accountCreation.html';
-  mailService.sendEmail(email, emailSubject, emailTemplate);
-  return user;
-};
+owasp.config({
+  minLength: 8,
+  minOptionalTestsToPass: 4,
+});
+
+// create user - for signup
+router.post('/create', async (req, res) => {
+  const {
+    username,
+    email,
+    password,
+    passwordConfirmation,
+    organization,
+    role,
+  } = req.body;
+
+  let result = owasp.test(password);
```

Comment:
Move this validation for the password matching and strength to the sign up component. Don't call the route until the validation is confirmed.